backend/trivia_api/views.py

```diff
@@ -61,6 +61,71 @@ def users(user_id):
     return dummy_users
 
 
+@blueprint.route("/signup", methods=["GET", "POST"])
+def signup_post():
+    if request.method == "POST":
+        # return jsonify({"message": "success"}, 200)
+        data = request.get_json("password")
+        newusers = data["username"]
+        #  newusers2 = str(newusers)
+        newpasswords = data["password"]
+        passwords = generate_password_hash(newpasswords, method="sha256")
+        newuser = User(username=newusers)
+        newpass = User(password=passwords)
+        if (
+            User.query.filter_by(username=newuser.username).first() is None
+            and User.query.filter_by(password=newpass.password).first() is None
+        ):
+            entry = User(username=newuser.username, password=newpass.password)
+            db.session.add(entry)
+            db.session.commit()
+            return jsonify({"code": 200, "message": "failed", "message": "success"})
```

Comment:
no need to pass code, esp if 200, also the format I already have in place is to use success: True or if failure, False along with an error field
```suggestion
            return {"success": True}
```